packages/run-protocol/test/supports.js

```diff
@@ -1,15 +1,17 @@
 // @ts-check
 /* global setImmediate */
 
+import { AmountMath } from '@agoric/ertp';
 import binaryVoteCounterBundle from '@agoric/governance/bundles/bundle-binaryVoteCounter.js';
 import committeeBundle from '@agoric/governance/bundles/bundle-committee.js';
 import contractGovernorBundle from '@agoric/governance/bundles/bundle-contractGovernor.js';
+import * as utils from '@agoric/vats/src/core/utils.js';
```

Comment:
If the grumble was for `*`, the use it to put them in the return of `setupBootstrap`. 
https://github.com/Agoric/agoric-sdk/blob/262097f7b82e2fba1a56b6f191b8ddbd4296ddaf/packages/run-protocol/test/supports.js#L90
Since it has no state, I think it would be better to import those functions directly when they're needed. But out of scope of course.

If the grumble was with imports moving, we could standardize using something like https://www.npmjs.com/package/organize-imports-cli

Reply:
It was the former. I agree that whoever is getting `*` from `setupBootstrap` should import it directly.  But `<grumble>` because I agree that we shouldn't fix it now.